internal/appconfig/serde.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml file generated for %s on %s\n\n", c.AppName, time.Now().Format(time.RFC3339))
```

Comment:
How about 
```
fly.toml app configuration file generated for mounts-per-group on 2023-04-17T22:03:13Z
```
to make it more obvious what `fly.toml` is for?

Reply:
sounds good to me. you rocks.